src/Server.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Swoole;
+
+use Swoole\Http\Server as SwooleHttpServer;
+
+class Server
+{
+
+    /**
+     * @var string
+     */
+    protected $host;
+
+    /**
+     * @var int
+     */
+    protected $port;
+
+    /**
+     * @var int
+     */
+    protected $mode;
+
+    /**
+     * @var int
+     */
+    protected $protocol;
+
+    /**
+     * @var array
+     */
+    protected $options = [];
+
+    /**
+     * @var SwooleHttpServer
+     */
+    protected $swooleServer;
+
+    public function __construct(string $host, int $port, int $mode, int $protocol, array $options)
+    {
+        $this->host = $host;
+        $this->port = $port;
+        $this->mode = $mode;
+        $this->protocol = $protocol;
```

Comment:
Once the `$mode` and `$protocol` values are documented, we should also have some checking here to ensure they are _correct_ values; I'm pretty sure not just ANY integer is allowed!